src/components/dictionaryConcepts/components/AsyncMulti.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react';
+import AsyncSelect from 'react-select/lib/Async';
+import { notify } from 'react-notify-toast';
+import PropTypes from 'prop-types';
+import instance from '../../../config/axiosConfig';
+
+const promiseOptions = async (q) => {
+  const url = `concepts/?q=${q}&limit=0&verbose=true`;
+  try {
+    const response = await instance.get(url);
+    const options = response.data.map(concept => ({ value: `${concept.url}`, label: `${concept.owner}: ${concept.display_name}` }));
+    return options;
+  } catch (error) {
+    notify.show('There is something wrong with your connection', 'error', 3000);
+  }
+  return false;
+};
+
+class AsyncMulti extends Component {
```

Comment:
This class name `AsyncMulti` should be renamed to another name that reveals its intention. Also, consider renaming the file name.

Reply:
@collinewait let me change the name.